.env

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)